### DISCUSSION TITLE
Where is the webhook api?

### DISCUSSION THREAD

**Discussion by siddharthroy12** [forem/forem#16747] [Q&A]
I'm trying to create a webhook using dev.to/api/webhooks but it's showing 404 error and there is no instruction in the docs

**Comment by nickytonline** [comment]
Webhooks were removed a while back. See #15827

**Comment by alsaaahalex** [comment]
Hi,
Alsa halex here,
I am also searching for webhook api. The documentation for dev[.](https://perfectclimbing.com/best-shoes-for-lead-climbing/)to/api/webhooks does not contain any instructions regarding how to create a webhook.
